# SMART ligation-free tools for sequencing coding and non-coding RNA from low input samples
Nathalie Bolduc*, Simon Lee, Michael Wu, and Andrew Farmer 
Takara Bio USA, Inc., Mountain View, CA 94043, USA 
# Abstract  
&#8195;&#8195;Small and long non-coding RNAs play a major role in the regulation of gene expression and disease. Obtaining an accurate portrait of their expression levels from small sample inputs carries potential for both the fulfillment of basic research objectives and the development of novel therapeutics and clinical diagnostic solutions. Towards this end, we have built upon the sensitivity of our SMART® technology to develop the SMARTer® Stranded Total RNA-Seq Kit - Pico Input Mammalian (Pico kit). The Pico kit relies on random priming to synthesize cDNA from both polyadenylated and non-polyadenylated RNA, thus ensuring the capture of the non-coding RNA fraction. In addition, we developed a novel technology, ZapR, which allows for the removal of ribosomal RNA (rRNA)-derived cDNA after reverse transcription and library amplification.While our original Pico kit could accommodate inputs ranging from 250 pg to 10 ng of purified total RNA, we sought to expand the capabilities of this technology to accommodate intact cells and improve the kit’s overall sensitivity and sequencing performance. While the Pico kit enables strand-specific transcriptome analysis from very low amounts of total RNA or cells, it does not allow for the analysis of small RNAs, such as miRNAs, piRNAs, etc. To address this limitation, we developed the SMARTer smRNA-Seq Kit for Illumina®, a novel, ligation-free approach for the preparation of small RNA sequencing libraries that leverages RNA 3’polyadenylation followed by cDNA synthesis and template switching. This approach minimizes sample representation bias and is sensitive enough to accommodate inputs of as little as 1 ng of total RNA. The combination of the SMARTer Stranded Total RNA-Seq Kit - Pico Input Mammalian and the SMARTer smRNA-Seq Kit for Illumina provides a complete toolkit for accurate, sensitive, and reproducible detection of small and long non-coding RNAs with fast and easy workflows.

![biao test](http://tianchen2019.cn3v.net/Clontech4.png)
Figure 1. Schematic of technology used by the SMARTer Stranded Total RNA-Seq Kit v2 - Pico Input Mammalian.
SMART technology is used in this ligation-free protocol to preserve strand-of-origin information. Random priming allows for the generation of cDNA from all RNA fragments in the sample, including rRNA. When SMARTScribe™ Reverse Transcriptase (RT) reaches the 5’ end of an RNA template, the enzyme’s terminal transferase activity adds a few non-templated nucleotides (shown asXs) to the 3’ end of the cDNA. The Pico v2 SMART Adapter base-pairs with the non-templated nucleotides, creating an extended template that enables the RT to continue cDNA synthesis and add a priming site for subsequent PCR amplification. In the next step,a first round of PCR amplification (PCR 1 ) adds full-length Illumina adapters, including barcodes. Ribosomal cDNA (originating from rRNA) is then cleaved by ZapR in the presence of mammalian-specific R-Probes. This process leaves library fragments originating from non-rRNA molecules untouched, with priming sites available on both 5’ and 3’ ends for further PCR amplification. Final libraries are compatible with sequencing on any Illumina platform.
![](http://tianchen2019.cn3v.net/Clontech5.png)
![](http://tianchen2019.cn3v.net/Clontech6.png)
# Conclusions
SMART technology is a very sensitive and versatile tool for NGS library preparation from coding and non-coding RNAs of all sizes.  
**Whole transcriptome/lncRNA analysis:**  
* SMARTer Stranded technology, included in the new SMARTer Stranded Total RNA-Seq Kit v2 - Pico Input Mammalian, is well-suited for RNA-seq applications that require exceptional sensitivity and strand-of-origin information, such as research involving lncRNAs.
* The new v2 workflow is very similar to that of the original SMARTer Stranded Total RNA-Seq Kit - Pico Input Mammalian, but provides a more user-friendly experience, and importantly, significantly higher performance on all Illumina sequencing platforms.  
**smRNA analysis:**
* Using RNA 3’ polyadenylation and SMART template-switching technology, we developed a small RNA library preparation method that provides greater accuracy than approaches involving adapter ligation.
* In addition to capturing miRNAs, the SMARTer smRNA-Seq Kit for Illumina enables analysis of piRNAs, snoRNAs, snoRNAs, snRNAs, etc.
![](http://tianchen2019.cn3v.net/Clontech7.png)
![](http://tianchen2019.cn3v.net/Clontech8.png)
![](http://tianchen2019.cn3v.net/Clontech9.png)
![](http://tianchen2019.cn3v.net/Clontech10.png)
